src/js/components/Values/IssuesByBallotItemDisplayList.jsx

```diff
@@ -155,7 +157,10 @@ class IssuesByBallotItemDisplayList extends Component {
     );
 
     return (
-      <Wrapper>
+      <Wrapper
+      onMouseEnter={this.props.handleLeave}
```

Comment:
Since handleLeave and handleHover are not required functions (which is correct), I would create a local function that wraps the incoming functions, and then call those onMouseEnter and onMouseLeave. For an example of this, search for goToCandidate and goToCandidateLinkLocal in WebApp/src/js/components/Widgets/BallotItemSupportOpposeCountDisplay.jsx This will fail much more gracefully if the handleLeave and handleHover functions aren't passed in.

Reply:
Just to clarify: 

In the IssuesByBallotItemDisplayList.jsx file, you want me to create a function inside render() that is called by the `onMouseEnter` on line 161, and through that function, access `this.props.handleLeave`?

Reply:
I would recommend naming the local function `handleLeaveHoverAreaLocal` and call that local function from line 161. I would recommend changing the incoming prop to be named `handleLeaveHoverArea` for clarity.